server/pbench/bin/pbench-package-tarballs

```diff
@@ -0,0 +1,85 @@
+#! /bin/bash
+
+# load common things
+opts=$SHELLOPTS
+case $opts in
+    *xtrace*)
+        dir=$(dirname $(which $0))
+        PROG=$(basename $(which $0))
+        ;;
+    *)
+        dir=$(dirname $0)
+        PROG=$(basename $0)
+        ;;
+esac
+
+remotearchive="/pbench/archive/fs-version-001"
```

Comment:
This should be just

    remotearchive=$ARCHIVE

but it has to come *after* the sourcing of pbench-base.sh. In general, there is a bunch of variables defined in the base script: you should use them, rather than hard-coding things in various scripts.